README.md

```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Truc Duong's portfolio and SPS projects
```

Comment:
nit - Keep the `.` at the end of the sentence.

Nice job! :)

Reply:
Oops! Thank you :D!